tests/TestCase/Shell/BakeShellTest.php

```diff
@@ -106,7 +106,7 @@ public function testMain()
             ->method('out')
             ->with($this->stringContains('The following commands'));
 
-        $this->Shell->expects($this->exactly(17))
+        $this->Shell->expects($this->exactly(18))
```

Comment:
I think it's about time this test was changed to be actually useful "this method calls out 18 times" is of little to no value.